net/lora-gateway-hal/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2019 Xue Liu <liuxuenetmail@gmail>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-gateway-hal
+PKG_VERSION:=5.0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
Absolutely not. Rename this to PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

Reply:
It is fixed.